packages/trader/src/App/Components/Elements/Media/__tests__/media.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MediaItem from '../media-item';
+import { MediaHeading, MediaDescription, MediaIcon } from 'App/Components/Elements/Media';
+
+const test_children = 'Test Children';
+const test_disabled = 'Interval Duration Disabled Light Icon';
+const test_enabled = 'Interval Duration Enabled Light Icon';
+const mock_props = {
+    disabled: jest.fn(() => <div>{test_disabled}</div>),
+    enabled: jest.fn(() => <div>{test_enabled}</div>),
+    id: 'test_id',
+    is_enabled: true,
+};
+
```

Comment:
I wrote all tests in one file, because testing components are tiny (a few lines of code)

Reply:
I think all test cases should be in separate files.
cc: @maryia-deriv

Reply:
Agreed.

Reply:
No problem, I'll separate them